mergify_engine/actions/utils.py

```diff
@@ -52,6 +66,26 @@ async def validate_bot_account(
     if required_permissions is None:
         required_permissions = ["admin", "write", "maintain"]
 
+    try:
+        bot_account = await ctxt.pull_request.render_template(bot_account_template)
+    except context.RenderTemplateFailure as rmf:
+        raise RenderBotAccountFailure(
+            check_api.Conclusion.FAILURE,
+            f"Invalid {option_name} template",
+            str(rmf),
+        )
+
+    try:
+        bot_account = typing.cast(
+            github_types.GitHubLogin, GitHubLoginSchema(bot_account)
+        )
+    except voluptuous.Invalid as e:
+        raise RenderBotAccountFailure(
+            check_api.Conclusion.FAILURE,
+            f"Invalid {option_name} template",
```

Comment:
```suggestion
            f"Invalid {option_name} value",
```
The template is ok but the value rendered is wrong.